packages/gp2-components/src/organisms/ProjectsHeader.tsx

```diff
@@ -0,0 +1,14 @@
+import { projectsImage } from '../images';
+import PageBanner from './PageBanner';
+
+const props = {
+  image: projectsImage,
+  position: 'center',
+  title: 'Project Directory',
+  description:
+    'Explore past and present GP2 projects being carried out by teams within the network and discover open opportunities to join them.',
+};
+
+const ProjectsHeader: React.FC = () => <PageBanner {...props} />;
```

Comment:
I wonder whats the advantage of having this as a separate component

Reply:
This is the same as working groups.